6-Typescript/common/interfaces/App.ts

```diff
@@ -0,0 +1,11 @@
+import DB from './DB'
```

Comment:
why App is in common ?

Reply:
because the jest tests need it and they're in the server folder